conf/messages.en

```diff
@@ -112,7 +112,7 @@ profit.change.hidden = Taxable profit
 distribution.title = Distributions
 distribution.heading = Distributions
 distribution.checkYourAnswersLabel = Distributions
-distribution.error.nonNumeric = Enter your Distributions using numbers
+sdistribution.error.nonNumeric = Enter your Distributions using numbers
```

Comment:
typo?

Reply:
This has been removed and merged with the below